mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb

```diff
@@ -248,12 +249,20 @@ func expandNetworkInterfaces(d TerraformResourceData, config *Config) ([]*comput
 			return nil, fmt.Errorf("cannot determine self_link for subnetwork %q: %s", subnetwork, err)
 		}
 
+		var nt string
+		if data["nic_type"] != nil {
```

Comment:
Usually this is broken out into another method, similar to below `AccessConfigs: expandAccessConfigs(data["access_config"].([]interface{})),`

You could do `NicType:       expandNicType(data["nic_type"].(interface{})),` and put the nic_type specific logic in there.